pydis_site/apps/content/resources/guides/pydis-guides/contributing/sir-lancebot.md

```diff
@@ -16,6 +16,15 @@ toc: 1
     - [MacOS Installer](https://git-scm.com/download/mac) or `brew install git`
     - [Linux](https://git-scm.com/download/linux)
 
+## Using Gitpod
+Sir Lancebot can be edited and tested on Gitpod. Gitpod will automatically install the correct dependencies and Python version, so you can get straight to coding.
+To do this, you will need a Gitpod account, which you can get [here](https://www.gitpod.io/#get-started). Afterwards, either click the button on Sir Lancebot's README or go to https://gitpod.io/#/python-discord/sir-lancebot and run the following commands in the terminal:
+```sh
+git remote rename origin upstream
+git add remote origin https://github.com/{your_username}/sir-lancebot
+```
+This will swap the Python Discord origin to upstream and add your own repository as the origin. Once you add your environment variables, you're ready to start contributing.
```

Comment:
I think that many starting out aren't that good with Git and won't know what upstream and origin means. Besides, it is only convention to name them like that.

Perhaps we can explain it more-so as parent and fork or similar?

Reply:
sure